core/utils/poly-look/src/themes.css

```diff
@@ -0,0 +1,9 @@
+.poly-theme-light {
+  --theme-primary-color: var(--color-dark);
```

Comment:
I wonder if we shouldn't prefix all those variables as well, given how there's only a global namespace for variables (AFAIK). But probably best done in a separate refactoring!

Reply:
I moved them to global, which I think makes sense. Would you still prefix them?